source/documentation/services/github/jml4-tool.html.md.erb

```diff
@@ -0,0 +1,53 @@
+---
+owner_slack: "#operations-engineering-alerts"
+title: Monthly Upload of Github User Data for JML4 Tool
+last_reviewed_on: 2024-01-05
+review_in: 3 months
+---
+
+# Monthly Upload of Github User Data for JML4 Tool
+
+## Overview
+
+This runbook relates to a trial of the [JML4 Tool](https://app.powerbi.com/groups/me/reports/ad061100-2417-4616-9481-efb533fb88d9/ReportSectionbe145fa8de8a4c99e949?experience=power-bi) (JML = Joiners, Movers, Leavers). The JML4 Tool is a user account remediation tool that collates user data from multiple sources to identify leavers.
+
+So that we can use this tool we provide the Data Controller with a monthly extract of Github user data.
+
+This runbook covers the data export from Github, and upload to Sharepoint processes. The Data Controller will process data from Sharepoint.
+
+## Frequency of process
+
+This process is run on the last working day of each month. Data must be uploaded within 5 working days of the last day of the month.
+
+## Github User Data Exports
+
+1. [Export a user list] from `Minsitry of Justice` organisation in `.csv` format
```

Comment:
May be good to specify what level of access is needed for this. Will the teams' current access be enough, or is this something that will be able to be performed by Enterprise Owners? 👀

Reply:
Have added some pre-requisites